docs/user-guide/ecosystem.md

```diff
@@ -0,0 +1,61 @@
+# Polars ecosystem
+
+## Introduction
+
+On this page you can find a non-exhaustive list of libraries and tools that support Polars. As the data ecosystem is evolving fast, more libraries will likely support Polars in the future. One of the main drivers is that Polars makes use of `Apache Arrow` in it's backend.
+
+#### [Apache Arrow](https://arrow.apache.org/)
+
+Apache Arrow enables zero-copy reads of data, meaning that data can be directly accessed in its in-memory format without the need for copying or serialisation. This greatly enhances performance, especially when sharing data between different processes or systems. Using Apache Arrow, Polars is compatible with a wide range of libraries that also make use of Apache Arrow, like Pandas and Pyspark.
```

Comment:
This is only possible within the same process without copying. It is true that there is no serializatin if two differen processes share arrow memory. 

> This greatly enhances performance

I would mostly say that it enhances performance when integrating with different tools.

Reply:
Thanks, I'll update in the next iteration